fe1-web/src/features/rollCall/screens/RollCallScanner.tsx

```diff
@@ -68,6 +71,8 @@ const RollCallOpened = () => {
   // is never changed, i.e. the bound variables stay the same during the scanner's lifetime
   // this is due to a react-camera bug and is hopefully fixed at some point in the future
   const attendeePopTokens = useRef(navigationAttendeePopTokens);
+  const [, updateState] = React.useState<unknown>();
+  const forceUpdate = React.useCallback(() => updateState({}), []);
```

Comment:
This is what I mentioned in the second meeting as a workaround for @dayan9265's test to pass since we use a `useRef` rather than a `useState` for the attendee's pop tokens due to https://github.com/dedis/popstellar/issues/1306

Reply:
Using a counter value would be the nicer solution but the scanner callback won't be updated which means

`const [counter, setCounter] = useState(0);
...
onScan={() => setCounter(counter + 1)}
`

would not work, we'd have to use the state from `useRef`

`const [counter, setCounter] = useState(0);
...
onScan={() => setCounter(scannedTokens.current.length + 1)}
`

Do you think this would be a better?

Reply:
Oh my bad, sorry. I didn't realize that you were talking about the counter, I thought it was about something else.
I don't think it would be much better because we would "force" having a state that does not what we want it to do, and pass the `useRef` state to it which seems weird to me. It is probably fine as it is.